### PR TITLE
Added topic groovy.

### PR DESCRIPTION
**Pull request by andrew-boutin**
- [X] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [ ] Suggesting edits to an existing Topic page
  - [X] Curating a new Topic page

***********EDITING AN EXISTING TOPIC PAGE************

I'm suggesting these edits to an existing topic:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

Please explain why these changes are necessary:

************CURATING A NEW TOPIC PAGE************

- [X] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [X] My folder contains a `*.png` image (if applicable) and `index.md`
- [X] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this Topic page should be curated:
Groovy is a very popular language with many tags on GitHub. I think out of all the things that could gets tags that programming languages should be the first.

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/groovy/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+aliases: groovy-lang
```
Comment (cheshire137) [inline topics/groovy/index.md]: `groovy-lang` isn't currently used on any public repositories, see https://github.com/topics/groovy-lang. Could you remove this line?

File: topics/groovy/index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+created_by: James Strachan
+display_name: Groovy
+logo: groovy.png
```
Comment (cheshire137) [inline topics/groovy/index.md]: The groovy.png image is currently bigger than the required 288x288 size. Can you resize it to fit?

File: topics/groovy/index.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+released: January 2, 2007
```
Comment (cheshire137) [inline topics/groovy/index.md]: Where did you get this release date? The Wikipedia page has 2003 as when Groovy first appeared.

You could add a `related: grails, java, language` line.

File: topics/groovy/index.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+short_description: Apache Groovy is an object-oriented programming language for the Java platform.
+topic: groovy
+url: http://groovy-lang.org/index.html
```
Comment (cheshire137) [inline topics/groovy/index.md]: How about just `http://groovy-lang.org`?

File: topics/groovy/index.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+wikipedia_url: https://en.wikipedia.org/wiki/Groovy_(programming_language)
+---
+Apache Groovy is a powerful, optionally typed and dynamic language, with static-typing and static compilation capabilities, for the Java platform aimed at improving developer productivity thanks to a concise, familiar and easy to learn syntax. It integrates smoothly with any Java program, and immediately delivers to your application powerful features, including scripting capabilities, Domain-Specific Language authoring, runtime and compile-time meta-programming and functional programming.
```
Comment (cheshire137) [inline topics/groovy/index.md]: The first sentence is pretty long, how about breaking it up? Maybe:

> Apache Groovy is a powerful, optionally typed and dynamic language for the Java platform. It features static typing and static compilation, as well as a concise, easy-to-learn syntax aimed at improving developer productivity.

Comment (cheshire137) [reply topics/groovy/index.md]: How about breaking up the second sentence and rephrasing a bit:

> It integrates smoothly with any Java program and provides scripting capabilities, Domain-Specific Language authoring, runtime and compile-time metaprogramming, and functional programming.

**Comment by MikeMcQuaid** [comment]
@andrew-boutin Gentle ping on the above comments.

File: topics/groovy/index.md (hunk 3 above)
Comment (andrew-boutin) [reply topics/groovy/index.md]: I was going with `Groovy 1.0 was released on January 2, 2007` from the wiki. I'll change it even though there's no mention of a "release" tied to the 2003 date.

File: topics/groovy/index.md (hunk 5 above)
Comment (andrew-boutin) [reply topics/groovy/index.md]: Will update. Was using descriptions from the groovy website itself.

**Comment by MikeMcQuaid** [comment]
> The groovy.png image is currently bigger than the required 288x288 size. Can you resize it to fit?

@andrew-boutin Can you ensure the aspect ration is preserved on resize? It now looks stretched. Thanks!

**Comment by MikeMcQuaid** [comment]
@andrew-boutin Shout if you're able to address this comments in future, thanks!

**Pull request closed without merging**